MarkBind/markbind
# Fix navbar overflow on mobile

**What is the purpose of this pull request?**
Fixes #1429

Also related:  #1699



**Overview of changes:**
- Set a minimum width for the scrollbar to be 8em. I tested this on the different device sizes of chrome and I think it looks fine on all; should guarantee at least least the whole placeholder attribute 'search' to be visible on the smallest screen sizes. 
- Make the section with items in the navigation bar scrollable in the case of overflow

![search_scroll](https://user-images.githubusercontent.com/59471320/153705539-92821248-d4f0-47c2-9794-48675ec7bc18.gif)
<img width="1009" alt="Screenshot 2022-02-12 at 5 52 51 PM" src="https://user-images.githubusercontent.com/59471320/153706448-3774bff2-93d9-4287-9bc7-663cf2d367cf.png">

**Anything you'd like to highlight / discuss:**
- @jonahtanjz since this modifies your previous changes for mobile, I'd like your opinion!
- Hiding the scrollbar in the navigation bar looks nice, but I'm not sure if it's very accessible. 
- Used em instead of vw for accessibility reasons(as it scales if the user zooms the screen)

**Testing instructions:**
Add lots of elements to the navigation bar to see it scroll + use google chrome mobile.


**Proposed commit message: (wrap lines at 72 characters)**
Make navbar scrollable & optimise search input size



---

**Checklist:** :ballot_box_with_check:
